src/helpers/utilities.ts

```diff
@@ -410,14 +410,53 @@ export const convertAmountToNativeDisplay = (
   return `${display} ${nativeSelected.symbol}`;
 };
 
+/**
+ * @desc convert from raw amount to rounded decimal
+ */
+export const convertRawAmountToRoundedDecimal = (
+  value: BigNumberish,
+  decimals = 18,
+  roundTo = 0
+): number => {
+  if (roundTo) {
+    const roundingFactor = 10 ** roundTo;
+    return (
+      Math.round(
+        new BigNumber(value)
+          .dividedBy(new BigNumber(10).pow(decimals))
+          .toNumber() * roundingFactor
+      ) / roundingFactor
+    );
+  } else {
+    return new BigNumber(value)
+      .dividedBy(new BigNumber(10).pow(decimals))
+      .toNumber();
+  }
+};
+
 /**
  * @desc convert from raw amount to decimal format
  */
 export const convertRawAmountToDecimalFormat = (
```

Comment:
These two utils are the same, no? Seems like we could keep `convertRawAmountToDecimalFormat` and then add a `roundBigNumber` or something like that that just handles rounding?

Reply:
one returns a string, the other a number. for my use case (nft pricing display) I thought string representation was unnecessary and kind of confusing. wdyt?